src/app/ar_assets/create/_components/StepContent/UploadQRCodeInsideImage.tsx

```diff
@@ -1,3 +1,41 @@
+import { IconUpload } from '@tabler/icons-react';
+import { useState } from 'react';
+import { Button, FileButton } from '@/shared/components/common/Button';
+import { Container } from '@/shared/components/common/Container';
+import { Image } from '@/shared/components/common/Image';
+import { Center, Group, Stack } from '@/shared/components/common/Layout';
+import { Text } from '@/shared/components/common/Text';
+import { Title } from '@/shared/components/common/Title';
+
 export const UploadQRCodeInsideImage = () => {
-  return <>QRコード内画像のアップロード</>;
+  const [file, setFile] = useState<File | null>(null);
+  return (
+    <Container>
+      <Title order={5} c="blue.6" mb={4}>
+        QRコード内画像
+      </Title>
+      <Text size="xs" c="gray.6" mb={28}>
+        QRコード真ん中に表示される画像です
+      </Text>
+      <Center>
+        <Stack>
+          <Image src="/QRcode.svg" alt="#" />
+          {file && (
+            <Text size="sm" ta="center" mt="sm" c={'gray.6'}>
```

Comment:
```suggestion
            <Text size="sm" ta="center" mt="sm" c="gray.6">
```
細かいけど、ここは""で統一した方が良さそう！